docs/trust-list-specification.md

```diff
@@ -0,0 +1,52 @@
+---
+sidebar_position: 4
+---
+
+# Trust List Specification
+
+## Introduction
+In order for data to be exchanged between different COVID-19 credential standards, there needs to be a means of unifying the trust lists of each respective specification. A Global Trust Network has the opportunity to bring existing and new local and regional trust networks together in a way that respects their designs and sovereignty while contributing to real interoperability among all of them.
+
+## Objective and Goals
+Finding alignment between existing trust list formats is the key to real interoperability. This can be done by defining a common trust list format to assemble and share public key infrastructure (PKI) for all COVID-19 credential specifications used by existing trust networks.
```

Comment:
```suggestion
Finding alignment between existing trust list formats is the key to real interoperability. This can be done by defining a common trust list format to assemble and share public key infrastructure (PKI) for all health credential specifications used by existing trust networks.
```

Reply:
Neelima reworded this section in her most recent commit, so that it now mentions just "credential specifications" instead of "COVID-19 credential specifications".